src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{" + "color='" + color + '\''
+                + ", number=" + number + '}';
+    }
+}
```

Comment:
Make sure to include access modifiers for the class and its members. In this case, the class and its members are package-private by default, which might not be intended. If you want to restrict access to the class only within the package, this is fine, otherwise consider adding public access modifiers.